### ISSUE TITLE
seekToDefaultPosition() not executed when LoopingMediaSource

### ISSUE THREAD

**Issue by fathallah1990** [google/ExoPlayer#5589]
ConcatenatingMediaSource concatenatingMediaSource = new ConcatenatingMediaSource();
        for (int i = 0; i <= suraLength; i++) {
            MediaSource mediaSource = buildMediaSource("url");
            LoopingMediaSource loopingMediaSource = new LoopingMediaSource(mediaSource, 4);
            concatenatingMediaSource.addMediaSource(loopingMediaSource);
        }
mp.seekToDefaultPosition(3);

seekToDefaultPosition() not executed when use  LoopingMediaSource

**Comment by tonihei** [comment]
That's indirectly caused by #5155 and not easily fixed until #5155 is resolved. A simple workaround is to not use `LoopingMediaSource`, but rather add the sources directly to the `ConcatenatingMediaSource` like that:
```
ConcatenatingMediaSource concatenatingMediaSource = new ConcatenatingMediaSource();
MediaSource mediaSource = buildMediaSource("url");
concatenatingMediaSource.addMediaSources(Collections.nCopies(4, mediaSource));
mp.seekToDefaultPosition(3);
```

**Comment by tonihei** [comment]
I'll close the issue as duplicate because it will be auto-fixed when #5155 is resolved and there is an easy workaround for this specific case.
